components/tooltip/__tests__/tooltip.test.js

```diff
@@ -49,4 +50,65 @@ describe('Tooltip', () => {
     expect(onVisibleChange.mock.calls.length).toBe(lastCount); // no change with lastCount
     expect(wrapper.ref('tooltip').prop('visible')).toBe(false);
   });
+
+  it('should hide when mouse leave native disabled button', () => {
+    const onVisibleChange = jest.fn();
+    const wrapper = mount(
+      <Tooltip
+        title="xxxxx"
+        mouseEnterDelay={0}
+        mouseLeaveDelay={0}
+        onVisibleChange={onVisibleChange}
+      >
+        <button disabled>Hello world!</button>
+      </Tooltip>
+    );
+
+    const button = wrapper.find('button').at(0);
+    button.simulate('mouseenter');
+    expect(onVisibleChange).not.toHaveBeenCalled();
+    expect(wrapper.ref('tooltip').prop('visible')).toBe(false);
```

Comment:
According to me, `Tooltip`'s visible should be true and `onVisibleChange` should be called when mouse enter `button`, right?